### ISSUE TITLE
Convert Russian comments to UTF-8 or re-write them in English

### ISSUE THREAD

**Issue by xplt** [KD-lab-Open-Source/qdEngine#1]
I took a quick glimpse at the code and I see that it uses CP1251 encoding for both comments and the code. For example,

Comments (in many-many places, these are just a few examples): 

- https://github.com/KD-lab-Open-Source/qdEngine/blob/main/src/QD/Core/System/app_core.cpp#L263
- https://github.com/KD-lab-Open-Source/qdEngine/blob/main/src/QD/Core/qdCore/qd_camera.cpp#L18
- https://github.com/KD-lab-Open-Source/qdEngine/blob/main/src/QD/Core/qdCore/qd_conditional_object.h#L10

Code: https://github.com/KD-lab-Open-Source/qdEngine/blob/main/src/QD/Core/System/app_error_handler.cpp#L15

I think, it should be either converted to UTF-8 (not sure if it's possible, if it is being used in code as well) or to be re-written in English

**Comment by ivan-ushakov** [comment]
Yes, converting to UTF-8 is preferable. Unfortunately some files have broken encoding, like `qdEngine/src/XLibs/Util/XMath/XMath.cpp`.

**Comment by cyber544** [comment]
А XMath и не собирается. Что делать? 
![photo_2024-12-16_01-38-50](https://github.com/user-attachments/assets/0b80be34-b596-4407-ab64-e55ec3d4a537)
